### ISSUE TITLE
Feature request: pause button should be disabled when playing from stream

### ISSUE THREAD

**Issue by HassuX** [rameplayerorg/rameplayer-defects#39]
Pause button should NOT be disabled when using stream:
- it does stop video, but you are not able to continue watching if you don't first stop video, so it would be better to disable pause button.

**Comment by fabled** [comment]
Agreed. Though, some streams are pauseable. Usually only live streams are not pauseable, and I don't think we can easily detect them. Perhaps disable pause for 'rtmp' streams for now? On the other hand, e.g. HLS (http) streams should be pauseable.
